enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -824,6 +825,12 @@ func (e *InProgressExecution) processOpUpdate(ctx context.Context, op *longrunni
 	if stage < e.lastStage {
 		return false, nil
 	}
+	// Log only on stage transitions or if it's been a while since we last
+	// logged.
+	if stage != e.lastStage || time.Since(e.lastLogTime) > 30*time.Second {
+		log.CtxInfof(ctx, "WaitExecution: %q in stage: %s", e.opName, stage)
```

Comment:
Did you mean to remove line 824?

Reply:
oops, yeah. I meant for this block to replace that line - moved it up